best-practices/tagging/index.html

```diff
@@ -176,7 +187,7 @@ <h4>Captions</h4>
 			</section>
 			<section id="em">
 				<h4>Emphasis</h4>
-
+				<p>Emphasis works differently in braille than it does in print in that the emphasized characters do not themselves change but instead additional characters are added that precede, and sometimes follow, the emphasized text. The following elements will not add the braille characters themselves but will instead just track where emphasis is placed so that reading systems can potentially offer features that utilize this information.</p>
```

Comment:
```suggestion
				<p>Emphasis works differently in braille than it does in print. Instead of changing the appearance of emphasized characters, additional characters are added that precede, and sometimes follow, the emphasized text. The following elements will not add the braille characters themselves but will instead just track where emphasis is placed so that reading systems can potentially offer features that utilize this information.</p>
```

Reply:
Fixed through responding to James' comments.